renovate.json

```diff
@@ -2,7 +2,7 @@
   "extends": ["config:base"],
   "packageRules": [
     {
-      "updateTypes": ["minor", "patch", "major"],
```

Comment:
We want to allow renovate to update packages to new minor or patch versions. But we don't want to include versions with braking changes